components/browser/menu2/src/main/java/mozilla/components/browser/menu2/BrowserMenuController.kt

```diff
@@ -56,7 +59,11 @@ class BrowserMenuController(
             view.onDismiss = ::dismiss
             view.onReopenMenu = ::reopenMenu
             setOnDismissListener(menuDismissListener)
-            displayPopup(view, anchor, orientation)
+            displayPopup(view, anchor, orientation, forceOrientation)
+
+            if (orientation == Orientation.UP && forceOrientation) {
+                view.scrollOnceToTheBottom()
+            }
```

Comment:
> Ok, I generally understand the problem and code fixes now. However, I am most unsure if we should scroll to the bottom when the menu is popped up. I imagine by default we will always lose out on seeing the "Settings" menu option from the scroll to the bottom since the user flow for opening the menu selector is through clicking on the menu selector when the keyboard is expanded and the toolbar is in edit mode.

Understood, however, this is to mirror the behaviour of a top down menu.  We show the first item always when it's in the down orientation, so when it's the up orientation should should do the same too.